glean-core/android/src/test/java/mozilla/telemetry/glean/private/AccumulationsBeforeGleanInitTest.kt

```diff
@@ -33,6 +34,8 @@ class AccumulationsBeforeGleanInitTest {
     @Before
     fun cleanup() {
         Glean.testDestroyGleanHandle()
+        @Suppress("EXPERIMENTAL_API_USAGE")
+        Dispatchers.API.setTaskQueueing(true)
```

Comment:
nit: can you add a comment (the one from the commit message is ok) before this line?

This should probably happen in `Glean.resetGlean` as well for other tests. Good catch. Would you mind filing a bug for it?

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1588463